dask/dataframe/tests/test_dataframe.py

```diff
@@ -2436,6 +2436,20 @@ def test_corr():
 
     pytest.raises(NotImplementedError, lambda: da.corr(db, method='spearman'))
     pytest.raises(TypeError, lambda: da.corr(ddf))
+    
+def test_corr_same_name():
```

Comment:
You'll need a newline above here to pass the flake8 tests. You can check your code passes beforehand as:

```
$ flake8 dask
```

Reply:
my bad, sorry. I thought I can quickly change it through the Github API but it's too prone to typos. I fixed it in https://github.com/dask/dask/pull/4934/commits/06dc7cccfdb18f82b7b503440ecaedaa6e7541ec